pkg/probe/http/request_test.go

```diff
@@ -38,3 +44,53 @@ func TestFormatURL(t *testing.T) {
 		}
 	}
 }
+
+func Test_userAgent(t *testing.T) {
+	tests := []struct {
+		name    string
+		purpose string
+		want    string
+	}{
+		{
+			name:    "test useragent output",
+			purpose: "agent",
+			want:    "kube-agent/.",
```

Comment:
I'm not sure if this test can flake in the future

```
func userAgent(purpose string) string {
	v := version.Get()
	return fmt.Sprintf("kube-%s/%s.%s", purpose, v.Major, v.Minor)
}

```

it depends on the output of `version.Get` , it seems is returning empty now ... this test doesn't looks worth adding it

Reply:
It has been removed, thx